packages/story-editor/src/app/media/utils/getResourceFromLocalFile.js

```diff
@@ -89,8 +89,8 @@ const getVideoResource = async (file) => {
     length = videoLength.length;
     lengthFormatted = videoLength.lengthFormatted;
   }
+  const hasAudio = hasVideoGotAudio(videoEl);
   const posterFile = await getFirstFrameOfVideo(src);
```

Comment:
We could do this as well. 
```suggestion
  await seekVideo(videoEl);
  const posterFile = await getImageFromVideo(videoEl);
```

Reply:
`getFirstFrameOfVideo` already calls `seekVideo`

Reply:
This is about reusing the video tag.